voltixApp/VoltixApp/Model/Coin.swift

```diff
@@ -123,7 +123,7 @@ class Coin: Codable, Hashable {
         let formatter = NumberFormatter()
         formatter.numberStyle = .currency
         formatter.locale = Locale.current
-        formatter.currencyCode = "USD"
+        formatter.currencySymbol = "$"
```

Comment:
A lot currency it not using `$` , for example `€` for Euro

Reply:
Fixed

Reply:
They are all dynamic now using the current currency